bench/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "sanctuary-type-classes-benchmarks",
+  "version": "0.0.0",
+  "private": true,
+  "license": "WTFPL",
```

Comment:
I believe we could instead add `sanctuary-type-classes@5.2.0` as a dev dependency in the top-level __package.json__.